egs2/chime4/enh_asr1/local/run_asr_pretraining.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+set -e
+set -u
+set -o pipefail
+
+train_set=
+valid_set=
+test_sets=
+asr_config=
+inference_config=
+lm_config=
+use_word_lm=
+word_vocab_size=
+
+. ../utils/parse_options.sh
+
+dir=$PWD
+cd ../../asr1
+
+./asr.sh                                   \
+    --stage 1 \
+    --stop_stage 11 \
```

Comment:
We often let the user to specify the stages by adding the argument `"$@"` at the end.

Reply:
I expect that users do not call `run_asr_pretraining.sh` directly (like other files in local). I assume that users will execute `run_multiiris.sh` that calls `run_enh_pretraining.sh` , `run_asr_pretraining.sh`, and `run_enh_asr_finetuning.sh`.
So, the default values are summarized in `run_multiiris.sh`.

If the name `run_xxx.sh` is misleading, I would like to rename them.